source/application/SXRMB/SXRMB.h

```diff
@@ -19,7 +19,9 @@
 namespace SXRMB {
 
 	enum SXRMBErrorCodes {
-		ErrorSXRMBBeamlineShuttersTimeout = 290101  // 2-CLS 9-SXRMB 01 module-01 errorID-01
+		ErrorSXRMBBeamlineShuttersTimeout = 290101,  // 2-CLS 9-SXRMB 01 module-01 errorID-01
```

Comment:
For the time being, let's use the old error code style (#defines in the header of the file). I want to get this sorted out, but I want to do it consistently.